README.md

```diff
@@ -4,7 +4,13 @@
 [Italiano](https://pnpm.io/it/) |
 [Português Brasileiro](https://pnpm.io/pt/)
 
-![](https://i.imgur.com/qlW1eEG.png)
+<h1>
```

Comment:
why is the picture in an h1 tag?

Reply:
It follows [markdownlint](https://github.com/DavidAnson/markdownlint#markdownlint).
> [`MD041` - First line in a file should be a top-level heading](https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md041.md)

(pnpm has language links, so it does not strictly follow this)

Reply:
There’s no need to add an h1 tag.

Reply:
OK, I've removed that ✂️ https://github.com/pnpm/pnpm/pull/8721/commits/2b1a984d438d44ae27377a672d64ef4110d2085f